package.json

```diff
@@ -82,5 +82,8 @@
   "resolutions": {
     "**/**/braces": "^2.3.1",
     "**/**/minimist": "^1.2.1"
+  },
+  "peerDependencies": {
+    "jsdom": "^16.2.2"
```

Comment:
I think it makes sense for this to be a peer rather than a full dependency.

Reply:
Hmm, potentially! I think that for now having it be a proper dependency is fine - and in the future if we have multiple render environment "plugins" then each plugin can have its own dependencies (specific versions of jsdom, etc.)

Reply:
I did this mostly so that we don't have to come here and update this repo, then update webapp just to make a simple JSDOM dep update for this stuff.